packages/example-studio/components/FunkyEditor/FunkyEditor.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import SlateEditor from '@sanity/form-builder/lib/inputs/BlockEditor-slate'
```

Comment:
I'm a bit sceptical to documenting this as the way to get the default block editor, as it makes refactoring the internals of the form-builder package a breaking change. IMO our official stance should be to never recommend importing anything from `./lib`

Reply:
Just realized that the BlockEditor is exported by default as `SlateInput` https://github.com/sanity-io/sanity/blob/54b0c2209cde2f9e8a70fbe1fe35f9530587b18d/packages/@sanity/form-builder/src/index.js#L12

If we change the exported name to BlockEditor, the import would be:
```js
import {BlockEditor} from '@sanity/form-builder/lib/inputs/BlockEditor-slate'
```

Reply:
That would be nice yes!

Reply:
Renamed in #210

Reply:
I just realized that @sanity/form-builder is not a direct dependency in the studio :(

Reply:
Fixed in #210 by exporting it from the sanity form builder part

Reply:
So the import would now be: ```js
import {BlockEditor} from 'part:@sanity/form-builder'
```